### DISCUSSION TITLE
Second drive for Rippled database - Not working

### DISCUSSION THREAD

**Discussion by JohnDoeOffline** [ripple/rippled#5158] [Q&A] [answered]
I have installed "Rippled" on my server (Ubuntu 24.04) on the main drive "/". With the original settings, Rippled runs fine.

I want the Rippled database running on another drive. Rippled config file "rippled.cfg" provides the following method to do excactly that:

	[node_db]
	type=RocksDB
	path=/media/nodes/ripple/db/nudb

	[database_path]
	/media/nodes/ripple/db

But when I restart the Rippled service:

	sudo systemctl daemon-reload
	sudo systemctl restart rippled.service
	sudo systemctl status rippled.service

I get this error:

	[...] systemd[1]: Started rippled.service - Ripple Daemon.
	[...] rippled[16375]: terminate called after throwing an instance of 'std::runtime_error'
	[...] rippled[16375]:   what():  Can not create "/media/nodes/ripple/db"

I have created all the necessary directories and changed their ownership. The 600GB SAS drive has a speed of 10K RPM, but I keep encountering this error.

PS: I am an absolute beginner to Ubuntu and Rippled.

...help...

**Comment by ximinez** [comment]
Can you post the output of `ls -laR /media/nodes/ripple`?

**Comment by JohnDoeOffline** [comment, marked as answer]
Hi. My problem was solved at UbuntuForums. The solution was adding the drive to the file  **/etc/fstab**  (ask chatGPT on howto).
